.pipelines/genmanifest-createpr-template.yaml

```diff
@@ -3,12 +3,12 @@ parameters:
     default: deploy/$(Build.BuildNumber)/$(MANIFESTS_BRANCH)
     type: string
   - name: MANIFEST_DIR
-    default: $(System.DefaultWorkingDirectory)/$(MANIFESTS_REPO)
+    default: $(System.DefaultWorkingDirectory)/$(MANIFEST_REPO)
```

Comment:
this should stay as MANIFESTS_REPO right?

Reply:
Yes